src/h5web/ConfigProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import type { ReactNode } from 'react';
+import { HeatmapConfigProvider } from './vis-packs/core/heatmap/config';
+import { LineConfigProvider } from './vis-packs/core/line/config';
+
+interface Props {
+  children: ReactNode;
+}
```

Comment:
This is where we'll add props demanding on what we decide in https://github.com/silx-kit/jupyterlab-h5web/issues/20